scripts/exgfs_wave_post_pnt.sh

```diff
@@ -604,96 +382,42 @@
 
   set_trace
 
-# 6.b Spectral data files
-
-  if [ ${USE_CFP:-"NO"} = "YES" ]; then nm=0; fi
+# 3.b Execute the taring
 
-  if [ ${USE_CFP:-"NO"} = "YES" ] && [ "$DOBLL_WAV" = "YES" ]; then
-    if [ "$DOBNDPNT_WAV" = YES ]; then
-      if [ "$DOSPC_WAV" = YES ]; then
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG ibp $Nb > ${WAV_MOD_TAG}_ibp_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-      fi
-      if [ "$DOBLL_WAV" = YES ]; then
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG ibpbull $Nb > ${WAV_MOD_TAG}_ibpbull_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG ibpcbull $Nb > ${WAV_MOD_TAG}_ibpcbull_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-      fi
-    else
-      if [ "$DOSPC_WAV" = YES ]; then
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG spec $Nb > ${WAV_MOD_TAG}_spec_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-      fi
-      if [ "$DOBLL_WAV" = YES ]; then
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG bull $Nb > ${WAV_MOD_TAG}_bull_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-        echo "$nm ${USHgfs}/wave_tar.sh $WAV_MOD_TAG cbull $Nb > ${WAV_MOD_TAG}_cbull_tar.out 2>&1 "   >> cmdtarfile
-        nm=$(( nm + 1 ))
-      fi
+  if [[ "${DOBNDPNT_WAV}" == "YES" ]]; then
+    if [[ "${DOSPC_WAV}" == "YES" ]]; then
+      echo "${USHgfs}/wave_tar.sh ${WAV_MOD_TAG} ibp ${Nb} 2>&1 | tee ${WAV_MOD_TAG}_ibp_tar.out" >> cmdtarfile
+    fi
+    if [[ "${DOBLL_WAV}" == "YES" ]]; then
+      echo "${USHgfs}/wave_tar.sh ${WAV_MOD_TAG} ibpbull ${Nb} 2>&1 | tee ${WAV_MOD_TAG}_ibpbull_tar.out" >> cmdtarfile
+      echo "${USHgfs}/wave_tar.sh ${WAV_MOD_TAG} ibpcbull ${Nb} 2>&1 | tee ${WAV_MOD_TAG}_ibpcbull_tar.out" >> cmdtarfile
```

Comment:
Only adding a comment here to confirm with @aerorahul. In regards to the open bugzilla #296 , is this an example you would need these to be handled (possibly all instances for wave jobs in a seperate PR? ) https://github.com/NOAA-EMC/global-workflow/blob/16219a3e6904662b774b3b23edc576b885737b3b/scripts/exgfs_wave_post_gridded_sbs.sh#L127

Reply:
@ I will resolve this conversation as @aerorahul has confirmed that was already is in the code addresses this issue. Thanks for clarification. I will apply the same fix to my PR.